src/main/java/edu/illinois/techservices/sympa/SympaClient.java

```diff
@@ -200,6 +200,198 @@ public static void createList(String cookie, String[] arguments) {
+      param4.addAttribute(new QName("xsi:type"), "xsd:boolean");
+      //param4.setValue("true");
+
+      soapMessage.saveChanges();
+
+      /*
+      System.out.println("Printing SOAP message?");
+      System.out.println(soapMessage.toString());
+      System.out.println("Done Printing SOAP message?");
+      */
+
+      // Send the SOAP message to the endpoint
+      SOAPMessage add = callSympaAPI(soapMessage);
+
+      System.out.println("\n add Response : ");
+      printSOAPMessage(add);
+
+    } catch(Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+
+  /**
+   * 
+   * @param cookie
+   */
+  public static void del(String cookie) {
+    try {
+
+      MessageFactory messageFactory = MessageFactory.newInstance();
+      SOAPMessage soapMessage = messageFactory.createMessage();
+      SOAPPart soapPart = soapMessage.getSOAPPart();
+      //String myNamespaceURI = "https://lists-dev.techservices.illinois.edu/lists/wsdl";
+      SOAPEnvelope envelope = addNamespaceDeclaration(soapPart);
+
+      MimeHeaders headers = soapMessage.getMimeHeaders();
+
+      //headers.addHeader("Authorization", encodedAuth);
+      headers.addHeader("Content-Type", "text/xml"); //application/soap+xml
+      headers.addHeader("SOAPAction", "urn:sympasoap#del"); 
+      headers.addHeader("Cookie", "sympa_session="+cookie);
+      SOAPBody soapBody = envelope.getBody();
+      
+      SOAPElement soapElement = soapBody.addChildElement("del", "ns", "urn:sympasoap");
+
+      System.out.println("Starting to add parameters");
+
+      SOAPElement param1 = soapElement.addChildElement("list", "ns");
+      param1.addTextNode("pbalesamplelist");
+      param1.addAttribute(new QName("xsi:type"), "xsd:string");
+
+      SOAPElement param2 = soapElement.addChildElement("email", "ns");
+      param2.addTextNode("rstanton@test.com");
+      param2.addAttribute(new QName("xsi:type"), "xsd:string");
+      
+      SOAPElement param3 = soapElement.addChildElement("quiet", "ns");
+      param3.addTextNode("0");
+      param3.addAttribute(new QName("xsi:type"), "xsd:boolean");
+      soapMessage.saveChanges();
+
+      System.out.println("Going to try to print soapMessage");
+      System.out.println(soapMessage.toString());
+      System.out.println("After trying to print soapMessage");
+
+      System.out.println("\n  Soap Call for del ");
+      printSOAPMessage(soapMessage);
+
+      SOAPMessage del = callSympaAPI(soapMessage);
+
+      System.out.println("\n del Response : ");
+      printSOAPMessage(del);
+
+    } catch(Exception e) {
+      e.printStackTrace();
+    }
+  }
+
+
+  /**
+   * 
+   * @param cookie
+   */
+  public static void getComplexLists(String cookie) {
+    try {
+
+      MessageFactory messageFactory = MessageFactory.newInstance();
+      SOAPMessage soapMessage = messageFactory.createMessage();
+      SOAPPart soapPart = soapMessage.getSOAPPart();
+      //String myNamespaceURI = "https://lists-dev.techservices.illinois.edu/lists/wsdl";
```

Comment:
Delete all the commented lines in all the methods (add, del, closeList and complexList) .